src/Misc/layoutroot/config.cmd

```diff
@@ -9,12 +9,29 @@ if defined VERBOSE_ARG (
 powershell.exe -NoLogo -Sta -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "$VerbosePreference = %VERBOSE_ARG% ; Get-ChildItem -LiteralPath '%~dp0' | ForEach-Object { Write-Verbose ('Unblock: {0}' -f $_.FullName) ; $_ } | Unblock-File | Out-Null ; Get-ChildItem -Recurse -LiteralPath '%~dp0bin', '%~dp0externals' | Where-Object { $_ -match '\.(ps1|psd1|psm1)$' } | ForEach-Object { Write-Verbose ('Unblock: {0}' -f $_.FullName) ; $_ } | Unblock-File | Out-Null"
 
 :run
-IF "%1"=="remove" (
-    "%~dp0bin\Agent.Listener.exe" unconfigure
-) ELSE (
-    "%~dp0bin\Agent.Listener.exe" configure %*
-    IF EXIST .Agent (attrib +h .Agent)
-    IF EXIST .Credentials (attrib +h .Credentials)
-    IF EXIST .Service (attrib +h .Service)
+
+IF "%~1" neq "remove" goto config
```

Comment:
I just chatted with @bryanmacfarlane to confirm. The plan is to go the opposite direction, Agent.Listener supports "remove", not "unconfigure". That will simplify both the `config.cmd` and `config.sh`.